deno/analyze.ts

```diff
@@ -2,11 +2,11 @@ import init, {
   source,
   tree as wasm_tree,
   analyze as wasm_analyze,
-} from "./wasm.js";
+} from "../wasm/wasm.js";
 
 await init(source);
 
-export function tree(filename: string, src: string) {
+export function extractDependencies(filename: string, src: string) {
```

Comment:
I think the name makes more sense

Reply:
`tree` is much shorter tho, let's keep it that way :smile: